tests/UnifiedSpecTests/Operation.php

```diff
@@ -248,17 +261,17 @@ private function executeForClient(Client $client)
 
         switch ($this->name) {
             case 'createChangeStream':
-                $changeStream = $client->watch(
-                    $args['pipeline'] ?? [],
+                assertArrayHasKey('pipeline', $args);
+                assertIsArray($args['pipeline']);
+
+                return $client->watch(
+                    $args['pipeline'],
                     array_diff_key($args, ['pipeline' => 1])
                 );
-                $changeStream->rewind();
```

Comment:
It's not clear to me why we were ever rewinding here. When a ChangeStream is first instantiated, its key is `null`. This means that `iteratorUntilDocumentOrError` is going to make an initial `rewind()` call on its own before attempting to check for `valid()` and either return the first result or loop with `next()` until we hit a `valid()` element.

Removing this didn't break any of the existing change stream tests, so I think it's safe to conclude that this was redundant.